examples/petsc/include/swarmutils.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
All files in the libCEED repo have this boilerplate at the top

```
// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
//
// SPDX-License-Identifier: BSD-2-Clause
//
// This file is part of CEED:  http://github.com/ceed

/// @file
/// libCEED QFunctions for mass operator example using PETSc
```

Reply:
Gotcha -- it seemed a little inconsistent in the other source/headers in the petsc examples